ai-library/base/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- crd.yaml
```

Comment:
Should we split CRDs to a separate component? e.g.

* ai-library/cluster - CRDs, potentially ClusterRoles etc. - i.e. cluster wide resources
* ai-library/operator (or something like that)

There is an issue on KF for that:https://github.com/kubeflow/manifests/issues/391

And for us it is especially important - in case we want to be enable ODH deployment by an ordinary user, we need to make sure the user can limit the list of components to things that do not require cluster admin (theoretically):)

Reply:
I suppose it's better to be in-line with kf best practices